### ISSUE TITLE
Replicating the data splits

### ISSUE THREAD

**Issue by aga-relation** [1edv/evolution#8]
Hi, I am trying to replicate the data splits quoted in the paper and having issues.

There is no seed in the [data processing script](https://github.com/1edv/evolution/blob/92c94b2ba3e0ef57ed1d5f0c25cd2291eec0dac2/manuscript_code/model/tpu_model/data_processing.ipynb) hence no way to replicate your train/valid/test splits. Instead, I am trying to connect the files mentioned in the processing script with the files in the [data repository](https://zenodo.org/record/4436477#.YxsGcezML0N) but there are no matching file names and the README doesn't explain it either. Could you please specify which data files were used for training, validation, evaluating on random sequences and evaluating on naturally-occurring sequences?

Thank you! :)

**Comment by 1edv** [comment]
Hi,

Thank you for your question. 
- https://zenodo.org/record/4436477/files/complex_media_training_data_Glu.txt?download=1 : This is the training data used for the complex media
- https://zenodo.org/record/4436477/files/defined_media_training_data_SC_Ura.txt?download=1 : This is the training data used for the defined media
- https://zenodo.org/record/4436477/files/Random_testdata_complex_media.txt?download=1 : This is the random test data used for the complex media
- https://zenodo.org/record/4436477/files/Random_testdata_defined_media.csv?download=1 : This is the random test data used for the defined media

Good luck!

**Comment by aga-relation** [comment]
Great, thank you! Could you please clarify which value in the `Random_testdata_defined_media.csv` corresponds to gene expression?
